src/Symfony/Component/VarExporter/Tests/VarExporterTest.php

```diff
@@ -67,10 +67,13 @@ public function provideFailingSerialization()
 
         yield [$a];
 
-        $a = [null, $h];
-        $a[0] = &$a;
+        // This test segfaults on the final PHP 7.2 release
+        if (\PHP_VERSION_ID !== 70234) {
+            $a = [null, $h];
+            $a[0] = &$a;
 
-        yield [$a];
+            yield [$a];
+        }
```

Comment:
I was able to reproduce the segfaults we caught on GHA on my dev machine (macOS 11.4 with PHP 7.2 from Homebrew). I don't know if we can do anything about them, but since nobody is going to patch PHP 7.2 for us anymore, I decided to skip the corresponding tests.

On Travis, we get an older version of PHP 7.2 (7.2.27). I assume that's why the segfault does not happen there.

Reply:
can't we stick to 7.2.27?

Reply:
We probably can (have to try out), but pinning that version just because Travis wasn't updated for some time feels a bit arbitrary, doesn't it?

Reply:
That'd be fine to me: we don't test patch level versions of php here. We just need a version that is capable of running the test suite.

Reply:
I tried to explicitly request `7.2.27`, but Setup PHP still seems to install the latest 7.2.

<img width="206" alt="Bildschirmfoto 2021-06-14 um 16 55 19" src="https://user-images.githubusercontent.com/1506493/121913036-58b0b780-cd31-11eb-9ea2-b2913a4444a7.png">

> We just need a version that is capable of running the test suite.

And apart from those two test cases, the version is capable. And the two cases that are excluded here, are still covered in the PHP 7.1/7.3 runs.